e2e/tsconfig.json

```diff
@@ -1,9 +1,8 @@
 {
+  "extends": "../tsconfig.shared",
```

Comment:
Do we want our e2e tests to share this? I don't think so, it's a very different setup.

Reply:
I did this because the current e2e/tsconfig.json doesn't work with newest typescript. if not extend, shall we instead just copy-paste everything from tsconfig.shared.json into e2e/tsconfig.json and just make sure to manually keep them up to date in the future?

Reply:
Only add the settings we need to add, not everything. Let's start minimal.